panel/src/components/Forms/Field/StructureField.vue

```diff
@@ -233,6 +234,10 @@ export default {
           model: this.endpoints.model
         };
 
+        if (this.autofocus === null && field.autofocus === true) {
+          this.autofocus = name;
+        }
```

Comment:
If there is more than one autofocus fields, should the first or last one be applied?

Reply:
It should normally be the first autofocus field.

Reply:
Okay, this code is already working on getting the first autofocus 👍